src/current-refinements/__tests__/__snapshots__/current-refinements.spec.ts.snap

```diff
@@ -62,7 +62,7 @@ exports[`CurrentRefinedValues renders markup with state 1`] = `
             <span
               class="ais-CurrentRefinements-label"
             >
-              Hierarchicalcategories.Lvl0:
+              Hierarchicalcategories.lvl0:
```

Comment:
Do you know why this has changed?

Reply:
yes, there has been some bug fixes in the titlecase pipe in Angular 6: https://github.com/angular/angular/blob/master/CHANGELOG.md#bug-fixes-39

I think the commit where this happened is here https://github.com/angular/angular/commit/7966744

No impact on us.